docs/Carcols.md

```diff
@@ -522,3 +529,110 @@ _
+ </Item>
+ ```
+
+ This is the meat and bones of the modkits. 
+  
+ | Tag Name | What is is? |  Accepted value type | Options
+|---|---|---|---|
+| modelName | This is the name of the file in the stream folder.  | Name | N/a | 
+| modShopLabel | This is the name that will show up in the customization screens. This is not really used in FiveM / GTA unless its being customized within a Los Santos Customs | Name | N/a |
+| linkedModels | This links two modkits together allowing you to turn two or more things on when you turn on option on. | Items | Items |
+| turnOffBones | This allows you to turn off bones within the car. For example, turning off a mirror when replacing the option. This has to be a misc item. | Item | Items |
+| type | This defines the type of the modkit. | Name | Find a list of types [here](https://gtamods.com/wiki/Carcols.ymt#type) |
+| bone | This defines what bone the modkit is linked to. Chassis is standard if you are just adding something on. | String | Find a list of bones [here](https://gtamods.com/wiki/Carcols.ymt#bone) |
+| collisionBone | This defines what collision the modkit item will use. Once again, the chassis is normally standard but you can add different collisions. | Name | Fin a lit of collisions [here](https://gtamods.com/wiki/Carcols.ymt#collisionBone) |
+| cameraPos | This allows you to change how the camera works with the modkit enables. This isn't normally changed from VMCP_DEFAULT though. | Name | N/a |
+| audioApply | This options isn't used very often alongside that there isn't a lot of information about it. It's best to leave the value at 1. | Name | N/a |
+| weight | The weight is what is applied to the vehicle once the modkit item is on. For example, if you add a massive wing onto your car you can add a bit more weight to slow the vehicle down. This option directly affects the handling. | String | N/a |
+| turnOffExtra | There isn't a lot of information on this but I'd assume it links with the vehicles.meta turnOffExtra.  | Boolean | true / false |
+| disableBonnetCamera | This option turns off the bonnet camera off when the modkit item in enabled in game. | Boolean | true / false |
+| allowBonnetSlide | This will stop players from sliding over the bonnet when the modkit item is enabled. | Boolean | true / false |
+  
+  
+### Debugging modkits
+  
+If in the event your modkits do not work, here are some tips on how to fix it:
+  
+*   Ensure your modkit ID at [Step 3](#step-3) is unique to the individual vehicle. The values can be from 0-1023 but some may overlap with other vehicles. Just keep changing the ID until your options show up for you. You do not need to restart the server to update this.
+*   The kiteName at [Step 3](#step-3) is not in the carvariations.meta. 
+*   The .yft modkit is not in the stream folder. Ensure they are all in there.
+*   The modelName is not the same as the name of the .yft modkit in the stream folder. Ensure they are both the same.
```

Comment:
This also falls into the "instructions" issue.

This could be just re-worded though. 
i.e. `Common Issues` (noun) instead of `Debugging` (verb)


For the bullets, most are good, but the first one is more instructional
> The .yft modkit is not in the stream folder. Ensure they are all in there.

which is good and states a possible issue versus

> Ensure your modkit ID at [Step 3](#step-3) is unique to the individual vehicle.

which gives instructions.
Hopefully this makes sense.